genesis/data.go

```diff
@@ -0,0 +1,45 @@
+package genesis
+
+import "math/big"
+
+// DelegationData specify the delegation address and the balance provided
+type DelegationData struct {
+	Address string `json:"address,omitempty"`
+	Balance string `json:"balance,omitempty"`
+	address []byte
+	balance *big.Int
+}
+
+// InitialBalance provides information about one entry in the genesis file
+type InitialBalance struct {
```

Comment:
Would this be better named `InitialAccount`? An account has balance, an address... etc

Reply:
done